content/en/real_user_monitoring/guide/monitor-browser-rum-data-for-electronjs-apps.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Monitor Browser RUM Data for Electron JS Apps 
+kind: guide
+beta: true
+description: Learn how monitor desktop applications that use Electron.js through the browser RUM SDK
+aliases:
+- /real_user_monitoring/guide/getting-started-feature-flags/
+further_reading:
+  - link: '/real_user_monitoring/'
+    tag: 'Documentation'
+    text: 'Learn about Real User Monitoring'
+---
+
+<div class="alert alert-warning">
+    Electron.js support for browser RUM is in beta.
+</div>
+
+## Overview
+
+Electron JS is an open source framework that can be used to build cross-platform macOS and Windows desktop applications.
+
+You can install and configure the Datadog Browser SDK to start monitoring applications built using Electron JS.
+
+## Installation
+
+To install the Datadog Browser SDK to support Electron JS apps:
+
+1. Set up and install [RUM Browser Monitoring][1], following the steps for CDN sync, CDN async, or npm.
+
+2. Set the `allowFallbackToLocalStorage` parameter to true in the RUM initialization configuration of each renderer process:
+
+   ```javascript
+   datadogRum.init({
+     applicationId: '<DATADOG_APPLICATION_ID>',
+     clientToken: '<DATADOG_CLIENT_TOKEN>',
+     site: '<DATADOG_SITE>',
+     ...
+     allowFallbackToLocalStorage: true
+     });
+   ```
+
+   **Note**: The Browser SDK only supports monitoring the *renderer processes* of an application. It does not initialize or monitor anything installed on the *main process*.
```

Comment:
@yannickadam how obvious would it be to developers what the main vs renderer processes are? Should we add more detail?

Reply:
It's probably Electron 101, but a link wouldn't hurt ! 
https://www.electronjs.org/docs/latest/tutorial/process-model#the-renderer-process

Reply:
Also - This **note** is pretty important, as it explains where to deploy the SDK. If I blindly follow Step 1 (setup RUM SDK), I'd probably install it inside my `main` process. Maybe we should mention it earlier (that we need to install within every renderer process)